misc/python/materialize/scalability/benchmark_executor.py

```diff
@@ -268,6 +272,17 @@ def execute_operation(
             df_details_cols.TRANSACTION_INDEX: transaction_index,
         }
 
+    def create_workload_instance(
+        self, workload_cls: type[Workload], endpoint: Endpoint
+    ) -> Workload:
+        workload = workload_cls()
+
+        if isinstance(workload, WorkloadWithContext):
+            workload.set_endpoint(endpoint)
+            workload.set_schema(self.schema)
```

Comment:
What are shemas used for?

Reply:
I needed it to set the database schema that is used by set up scripts because otherwise the table (of `SelectCount()`) was not found. It is technically no longer needed when replacing `SelectCount` with `SelectOne`.

Reply:
ACK